herbert_core/classes/instrument_classes/instrument/@IX_experiment/IX_experiment.m

```diff
@@ -343,9 +343,12 @@
             old_fldnms = {'filename','filepath','efix','emode','en','cu',...
                 'cv','psi','omega','dpsi','gl','gs','uoffset','u_to_rlu'};
             obj = IX_experiment();
+            obj.do_check_combo_arg = false;
```

Comment:
please explain in comments around this code why the checks need to be turned off at this poiint